### DISCUSSION TITLE
Share location and peoples with family

### DISCUSSION THREAD

**Discussion by DDimly** [immich-app/immich#11660] [General]
Hello
May be i bad in search and this quesion was already

I use Immich at home with my family
I upload all pfotos - and share all albums but my wife can not see locations and peolpes
Empty in url /people and /map and info in every photo has not location and peoples data
May be you can add switch for change mode on server to Family with auto share locations/peoples/albums and photos outside albums
Also it will enable global deduplication in upload process

Oh... too difficult, i think

**Comment by C-Otto** [comment]
See #12614
